capabilities_matrix/_topics/container_providers.md

```diff
@@ -2,27 +2,27 @@
-| Chargeback by Usage                     | ✅             | ✅            | ✅         | ❌                   |❌          |❌         |❌          |❌             |❌             |
-| Remote Cockpit Node Access              | ✅             | ❌            | ❌         | ❌                   |❌          |❌         |❌          |❌             |❌             |
-| Container Node Compliance Enforcement   | ✅             | ✅            | ✅         | ✅                   |✅          |✅         |✅          |✅             |✅             |
-| Container Node Policy Enforcement       | ✅             | ✅            | ✅         | ✅                   |✅          |✅         |✅          |✅             |✅             |
-| Replicator Compliance Enforcement       | ✅             | ✅            | ✅         | ✅                   |✅          |✅         |✅          |✅             |✅             |
-| Replicator Policy Enforcement           | ✅             | ✅            | ✅         | ✅                   |✅          |✅         |✅          |✅             |✅             |
-| Pod Compliance Enforcement              | ✅             | ✅            | ✅         | ✅                   |✅          |✅         |✅          |✅             |✅             |
-| Pod Policy Enforcement                  | ✅             | ✅            | ✅         | ✅                   |✅          |✅         |✅          |✅             |✅             |
-| Image Compliance Enforcement            | ✅             | ✅            | ✅         | ✅                   |✅          |✅         |✅          |✅             |✅             |
-| Image Policy Enforcement                | ✅             | ✅            | ✅         | ✅                   |✅          |✅         |✅          |✅             |✅             |
-| Template Provisioning                   | ✅             | ❌            | ✅         | ❌                   |❌          |❌         |❌          |❌             |❌             |
-| Smart State                             | ✅             | ❌            | ❌         | ❌                   |❌          |❌         |❌          |❌             |❌             |
-| OpenSCAP Execution and Report           | ✅             | ❌            | ✅         | ❌                   |❌          |❌         |❌          |❌             |❌             |
-| Cross Linking Underlying Infrastructure | ✅             | ✅            | ✅         | ✅                   |✅          |✅         |✅          |✅             |✅             |
-| Tag Mapping from Provider               | ✅             | ✅            | ✅         | ✅                   |✅          |✅         |✅          |✅             |✅             |
-| Tag Mapping to Provider                 | ❌             | ❌            | ❌         | ❌                   |❌          |❌         |❌          |❌             |❌             |
+| Feature                                 |  OpenShift 4.x | Kubernetes | Amazon EKS           | Google GKE | Azure AKS | Oracle OKE | IBM Cloud IKS | VMware Tanzu |
```

Comment:
You can _probably_ remove the 4.x bit as well since we don't put versions for others.  If we want to set a minimum we can probably do it earlier in the doc when we list out the providers.  Up to you.

Reply:
:+1: yeah good point, I'll do this when I follow-up with the changes for the token / metrics setup